.classpath

```diff
@@ -2,25 +2,28 @@
 <classpath>
 	<classpathentry kind="src" output="target/classes" path="src/main/java">
 		<attributes>
-			<attribute name="optional" value="true"/>
-			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="optional" value="true" />
+			<attribute name="maven.pomderived" value="true" />
```

Comment:
I understand that formatting may be incorrect, however such fixes require separate commit.
It makes tracking significant changes (i.e. myFigureJoe) more difficult.